tests/e2e/c/hashing_contract.go

```diff
@@ -1,11 +0,0 @@
-// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-// AUTOMATICALLY GENERATED. DO NOT EDIT!
-// Generated from hashing.sol by compile-contract.sh
```

Comment:
This file doesn't seem to have ever been checked in to the repo (even going back to when this file was originally added to the kurtosis e2e tests).

Reply:
I didn't actually use a CLI/golang tool to generate this... I Just used remix (browser app)...

I don't think it's worthwhile to work to setup CI to compile solidity code unless we start writing a bunch of non-trivial solidity and issuing it in CI.

Although the Interop team may already do this to check the teleporter contracts